rosdep/base.yaml

```diff
@@ -1462,6 +1469,7 @@ libconsole-bridge-dev:
   arch: [console-bridge]
   debian: [libconsole-bridge-dev]
   fedora: [console-bridge-devel]
+  freebsd: [ros-console_bridge]
```

Comment:
Since this appears to be a speculative key rather than one representing an existing port, do you mean to mix dashes and underscores in this port name? I know there's no restriction on doing so but it seems the common convention is to use dashes as the standard separator.

Reply:
This is actually in process as I've submitted a [new port for this](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=224896). I agree the port name is not great, but I'm hoping that this is something that can be discussed/changed when the port is actually committed. I'll be following this and sending an update as soon as I know more.

Reply:
Makes sense. Thanks for clarifying.